packages/ui-components/src/assets/svg-symbols.svg

```diff
@@ -1034,6 +1034,15 @@
 		<path d="M4 20h16v2H4v-2zM4 2h16v2H4V2zm9 7h3l-4-4-4 4h3v6H8l4 4 4-4h-3V9z"/>
 	</symbol>
 
+	<symbol id="kv-dashboard-custom"
```

Comment:
Missing add to the list (`packages/react-ui-components/src/stories/foundation/SvgIcons/SvgIcons-list.tsx`) to be visible in the SVG story

Reply:
Fixed 👍

Reply:
Also done